src/frontend/src/routes/(terms-and-conditions)/terms-and-conditions/+page.ts

```diff
@@ -0,0 +1,5 @@
+import { loadRouteParams, type RouteParams } from '$lib/utils/nav.utils';
```

Comment:
I would use the same `page.ts` as for the signer route because those parameters are not used here

Reply:
yes, I thought about it, but I was thinking that maybe at some point there will be a link inside the logged in page. So, if the user go "back", it would need the parameters

But that we can see when we cross that bridge! Changed

(and opened a new PR for the route)